torch/fx/experimental/sym_node.py

```diff
@@ -123,7 +123,7 @@ def compute_hint():
                 "Cannot create SymNode of type "
                 f"{pytype} with incompatible hint of type {type(hint)}"
             )
-            if self.shape_env._translation_validation_enabled:
+            if self.shape_env and self.shape_env._translation_validation_enabled:
```

Comment:
Why can shape_env be None here now?

Reply:
I think that was debug cruft - I'll remove in a follow-up PR.